crane-cloud/frontend
# Add Database Size

# Description
This PR adds Database size and fixes an error with the app list sort error.

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID
https://trello.com/c/1j88HRCL

## How Can This Be Tested?
Pull this branch locally and check out the projects page, the apps page, and also the database size on the database setting page

## Checklist:


# Screenshots
![Screenshot from 2021-08-05 19-40-44](https://user-images.githubusercontent.com/32802973/128390860-38bbe1aa-3531-4330-b9df-a3ed38a911fa.png)
